app/src/main/java/org/projectbuendia/client/models/AppModel.java

```diff
@@ -301,22 +301,13 @@ public FetchTypedCursorAsyncTask(
         }
 
         @Override protected TypedCursor<T> doInBackground(Void... voids) {
-            Cursor cursor = null;
-            try {
-                cursor = mContentResolver.query(
-                    mContentUri,
-                    mProjection,
-                    mFilter.getSelectionString(),
-                    mFilter.getSelectionArgs(mConstraint),
-                    null);
-
+            try (Cursor cursor = mContentResolver.query(
+                mContentUri,
+                mProjection,
+                mFilter.getSelectionString(),
+                mFilter.getSelectionArgs(mConstraint),
+                null)) {
```

Comment:
I just noticed this change made the patient list stop working. Probably because it closes the cursor before returning it.
Is that right? How can we avoid this?